### ISSUE TITLE
create list that mentors need to ensure students have done in week 1

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/master-reference#440]
These problems currently get solved in a really ad hoc way

+ @m4v15 @mattlub have highlighted the need to get students onto linux before the course really kicks into gear

+ [FAC11 stop go continue](https://github.com/foundersandcoders/london-curriculum/blob/master/stop-go-continue/fac-11/cohort-week-1.md#stop) - @morkeltry about ES Lint
    > Individuals getting left to play catch-up, because they need to resolve technology problems (Tom)
    >  + Git isn't properly set up.
    >  + Swapping computers, so that there is an equal divide of commits.
    >  + Tom has an EsLint problem (only one).
    >  + Help people set up EsLint.

+ issues in FAC9, FAC10, FAC11, FACN1 & FACN2 where students' git contributions weren't showing against their username (email not set up correctly?)  
https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/#you-havent-added-your-local-git-commit-email-to-your-profile

+ SSH

**Comment by mattlub** [comment]
what's the status of ESLint/general linting wrt the syllabus/master ref?

**Comment by jsms90** [comment]
> It's interesting to see that everyone in FAC11 is using it by week 1

Because it's been put in the [installation list](https://github.com/foundersandcoders/master-reference/tree/master/coursebook/precourse#installation-list).

But there's been a lot of talk in FAC11 (as you can see) about not knowing how to set it up. And I also see it as potentially problematic that different people might choose different configurations, rather than having a consistent for your project teams, because we offer no guidance on configuration.

**Comment by oliverjam** [comment]
Yeah if we're going to have ESLint on the installation list we should probably go over it in week 1 to clarify how to get the `.eslintrc` etc set up.
